user_funcs.py

```diff
@@ -116,3 +117,246 @@ def login(conn) -> tuple[str, int]:
+    DELETE FROM moviecollection 
+    WHERE collectionid = %s
+    """
+    # Deletes all movie IDs associated with the collection ID
+    delete_all_movie_query = """
+    DELETE FROM incollection
+    WHERE collectionid = %s
+    """
+
+    # Changes the name of the collection
+    change_name_query = """
+    UPDATE moviecollection
+    SET name = %s
+    WHERE collectionid = %s
+    """
+
+    
+    collection_ID = ""
+    movie_ID = movie_funcs.browse_movies(conn)
+    if movie_ID == -1:
+        print(f"Function is end, couldn't find movie")
+        return
+    with conn.cursor() as curs:
+        action = ""
+        curs.execute(query, (user_ID,))
+        collection_ID_list = curs.fetchall()
+        # User Select which collection they want to modify based off of Index
+        print(f"Here are all of the collections for the user\n{collection_ID_list}")
+        index = int(input("Please input a number that is within the index of the list of collection IDs for the collection you want: "))
+        if index >= len(collection_ID_list) or index < 0:
+            print(f"Invalid index, ending function...")
+            return
+
+        # Start of Actions
+        print("what action would you like to do?")
+        action = int(input_utils.get_input_matching("1 - remove a movie\n2 - add a movie \n3 - modify name of collection\n4 - delete collection\n5 - exit function: "))
+        if action == 1 or action == 2 or action == 3 or action == 4 or action == 5:
+             match action:
+                # action for removing a movie
+                # This works
+                case 1:
+                    curs.execute(remove_movie_query, (movie_ID, collection_ID_list[index][0]))
+
+                    # action for adding a movie
+                    # This does work
+                case 2:
+                    curs.execute(add_movie_query, (movie_ID, collection_ID_list[index][0]))
+
+                    # action to rename a collection
+                    # Fucntionality works
+                case 3:
+                    new_name = input("what would you like to name your collection: ")
+                    curs.execute(change_name_query, (new_name, collection_ID_list[index],))
+
+                    # action to delete a collection
+                case 4:
+                    #This does work
+
+                    curs.execute(delete_all_movie_query, (collection_ID_list[index],))
+                    curs.execute(delete_moviecollection_query, (collection_ID_list[index],))
+                case 5:
+                    return
+    conn.commit()
+    return
+
+def browse_collections(conn, user_ID) -> None:
+    """
+    A different version to print all information of a users collection/
+    list of collections
+
+    :param conn: Connection to the database
+    :param user_ID: The ID of the user
+    :return: Nothing
+    """
+     # A Query to get the collection ID for a certain User ID
+    query = """
```

Comment:
```suggestion
    query = """
```
Use a more descriptive name for this query.